packages/workspace/src/core/affected-project-graph/locators/implicit-json-changes.ts

```diff
@@ -0,0 +1,46 @@
+import { NxJson } from '../../shared-interfaces';
+import { FileChange } from '../../file-utils';
+import { JsonValueDiff } from '../../../utils/json-diff';
+
+export function getImplicitlyTouchedProjectsByJsonChanges(
+  workspaceJson: any,
+  nxJson: NxJson,
```

Comment:
For later:

I find this signature weird. Shouldn't this get the list of project nodes?